app/src/main/java/com/tip/lunchbox/view/activity/RestaurantDetails.java

```diff
@@ -0,0 +1,52 @@
+package com.tip.lunchbox.view.activity;
+
+import android.graphics.Color;
+import android.os.Bundle;
+import android.widget.ArrayAdapter;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.lifecycle.ViewModelProvider;
+
+import com.bumptech.glide.Glide;
+import com.tip.lunchbox.databinding.ActivityRestaurantDetailsBinding;
+import com.tip.lunchbox.model.Restaurant;
+import com.tip.lunchbox.viewmodel.RestaurantDetailsViewModel;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class RestaurantDetails extends AppCompatActivity {
+    RestaurantDetailsViewModel viewModel;
+    ActivityRestaurantDetailsBinding binding;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        binding = ActivityRestaurantDetailsBinding.inflate(getLayoutInflater());
+        setContentView(binding.getRoot());
+        viewModel = new ViewModelProvider(this).get(RestaurantDetailsViewModel.class);
+        String resId = getIntent().getStringExtra("ResID");
```

Comment:
Instead of directly using the identifier (_ResID_)r, you can create a constant inside the [Constants.java](https://github.com/devansh-299/LunchBox/blob/dev/app/src/main/java/com/tip/lunchbox/utilities/Constants.java) and use it here